account_invoice_tax_wizard/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Account Invoice Tax Wizard
+==========================
+
+This module adds a wizard to add manual taxes on invoices without
+such taxes are in each of the lines of the invoices.
```

Comment:
Dejar:
This module adds a wizard to add manual taxes on invoices without needing such taxes to be added in each of the invoice lines.